From 12c0a23476f5725b9fcf1eb1567823afac406cd1 Mon Sep 17 00:00:00 2001
From: lordgeorg <lordgeorg@googlemail.com>
Date: Fri, 1 Sep 2017 21:48:04 +0200
Subject: [PATCH] Sets version to 9999

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 33b05fa..273e786 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,11 +47,11 @@ set(META_PROJECT_DESCRIPTION "C++ Locator Library")
 set(META_AUTHOR_ORGANIZATION "CG Internals GmbH")
 set(META_AUTHOR_DOMAIN       "https://github.com/cginternals/cpplocate/")
 set(META_AUTHOR_MAINTAINER   "opensource@cginternals.com")
-set(META_VERSION_MAJOR       "1")
+set(META_VERSION_MAJOR       "9999")
 set(META_VERSION_MINOR       "0")
 set(META_VERSION_PATCH       "0")
 set(META_VERSION_REVISION    "${GIT_REV}")
-set(META_VERSION             "${META_VERSION_MAJOR}.${META_VERSION_MINOR}.${META_VERSION_PATCH}")
+set(META_VERSION             "9999")
 set(META_NAME_VERSION        "${META_PROJECT_NAME} v${META_VERSION} (${META_VERSION_REVISION})")
 set(META_CMAKE_INIT_SHA      "6a6f30b38a1cee31ccac3f091816f28e0f6bce4b")
 
-- 
2.16.2

