diff --git a/cmake/FindGLM.cmake b/cmake/FindGLM.cmake
new file mode 100644
index 0000000..8ee86a4
--- /dev/null
+++ b/cmake/FindGLM.cmake
@@ -0,0 +1,23 @@
+
+# GLM_FOUND
+# GLM_INCLUDE_DIR
+
+include(FindPackageHandleStandardArgs)
+
+FIND_PATH(GLM_INCLUDE_DIR glm/glm.hpp
+
+    PATHS
+    $ENV{GLM_DIR}
+    /usr
+    /usr/local
+    /sw
+    /opt/local
+
+	PATH_SUFFIXES
+    /include
+
+    DOC "The directory where glm/glm.hpp resides.")
+    
+find_package_handle_standard_args(GLM REQUIRED_VARS GLM_INCLUDE_DIR)
+
+mark_as_advanced(GLM_INCLUDE_DIR)
