diff --git a/CMakeLists.txt b/CMakeLists.txt
index ea84727..f079b66 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -130,8 +130,8 @@ if(UNIX AND SYSTEM_DIR_INSTALL)
     set(INSTALL_CMAKE     "share/${project}/cmake") # /usr/[local]/share/<project>/cmake
     set(INSTALL_EXAMPLES  "share/${project}")       # /usr/[local]/share/<project>
     set(INSTALL_BIN       "bin")                    # /usr/[local]/bin
-    set(INSTALL_SHARED    "lib")                    # /usr/[local]/lib
-    set(INSTALL_LIB       "lib")                    # /usr/[local]/lib
+    set(INSTALL_SHARED    "lib64")                    # /usr/[local]/lib
+    set(INSTALL_LIB       "lib64")                    # /usr/[local]/lib
     set(INSTALL_INCLUDE   "include")                # /usr/[local]/include
     set(INSTALL_DOC       "share/doc/${project}")   # /usr/[local]/share/doc/<project>
     set(INSTALL_SHORTCUTS "share/applications")     # /usr/[local]/share/applications
@@ -143,8 +143,8 @@ else()
     set(INSTALL_CMAKE     "cmake")                  # ./cmake
     set(INSTALL_EXAMPLES  ".")                      # ./
     set(INSTALL_BIN       ".")                      # ./
-    set(INSTALL_SHARED    "lib")                    # ./lib
-    set(INSTALL_LIB       "lib")                    # ./lib
+    set(INSTALL_SHARED    "lib64")                    # ./lib
+    set(INSTALL_LIB       "lib64")                    # ./lib
     set(INSTALL_INCLUDE   "include")                # ./include
     set(INSTALL_DOC       "doc")                    # ./doc
     set(INSTALL_SHORTCUTS "misc")                   # ./misc
