diff --git a/cmake/FindGMock.cmake b/cmake/FindGMock.cmake
new file mode 100644
index 0000000..9a2a865
--- /dev/null
+++ b/cmake/FindGMock.cmake
@@ -0,0 +1,68 @@
+# From https://github.com/triglav/cmake-findgmock/blob/master/FindGMock.cmake (07/06/2019)
+
+function(_gmock_append_debugs _endvar _library)
+  if(${_library} AND ${_library}_DEBUG)
+    set(_output optimized ${${_library}} debug ${${_library}_DEBUG})
+  else()
+    set(_output ${${_library}})
+  endif()
+  set(${_endvar} ${_output} PARENT_SCOPE)
+endfunction()
+
+function(_gmock_find_library _name)
+  find_library(${_name}
+    NAMES ${ARGN}
+    HINTS
+      $ENV{GMOCK_ROOT}
+      ${GMOCK_ROOT}
+    PATH_SUFFIXES ${_gmock_libpath_suffixes}
+  )
+  mark_as_advanced(${_name})
+endfunction()
+
+
+if(NOT DEFINED GMOCK_MSVC_SEARCH)
+  set(GMOCK_MSVC_SEARCH MD)
+endif()
+
+set(_gmock_libpath_suffixes lib)
+if(MSVC)
+  if(GMOCK_MSVC_SEARCH STREQUAL "MD")
+    list(APPEND _gmock_libpath_suffixes
+      msvc/gmock-md/Debug
+      msvc/gmock-md/Release)
+  elseif(GMOCK_MSVC_SEARCH STREQUAL "MT")
+    list(APPEND _gmock_libpath_suffixes
+      msvc/gmock/Debug
+      msvc/gmock/Release)
+  endif()
+endif()
+
+find_path(GMOCK_INCLUDE_DIR gmock/gmock.h
+  HINTS
+    $ENV{GMOCK_ROOT}/include
+    ${GMOCK_ROOT}/include
+)
+mark_as_advanced(GMOCK_INCLUDE_DIR)
+
+if(MSVC AND GMOCK_MSVC_SEARCH STREQUAL "MD")
+  _gmock_find_library(GMOCK_LIBRARY            gmock-md  gmock)
+  _gmock_find_library(GMOCK_LIBRARY_DEBUG      gmock-mdd gmockd)
+  _gmock_find_library(GMOCK_MAIN_LIBRARY       gmock_main-md  gmock_main)
+  _gmock_find_library(GMOCK_MAIN_LIBRARY_DEBUG gmock_main-mdd gmock_maind)
+else()
+  _gmock_find_library(GMOCK_LIBRARY            gmock)
+  _gmock_find_library(GMOCK_LIBRARY_DEBUG      gmockd)
+  _gmock_find_library(GMOCK_MAIN_LIBRARY       gmock_main)
+  _gmock_find_library(GMOCK_MAIN_LIBRARY_DEBUG gmock_maind)
+endif()
+
+include(FindPackageHandleStandardArgs)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(GMock DEFAULT_MSG GMOCK_LIBRARY GMOCK_INCLUDE_DIR GMOCK_MAIN_LIBRARY)
+
+if(GMOCK_FOUND)
+  set(GMOCK_INCLUDE_DIRS ${GMOCK_INCLUDE_DIR})
+  _gmock_append_debugs(GMOCK_LIBRARIES      GMOCK_LIBRARY)
+  _gmock_append_debugs(GMOCK_MAIN_LIBRARIES GMOCK_MAIN_LIBRARY)
+  set(GMOCK_BOTH_LIBRARIES ${GMOCK_LIBRARIES} ${GMOCK_MAIN_LIBRARIES})
+endif()
diff --git a/source/tests/CMakeLists.txt b/source/tests/CMakeLists.txt
index d3fd982..8af378b 100644
--- a/source/tests/CMakeLists.txt
+++ b/source/tests/CMakeLists.txt
@@ -56,6 +56,12 @@ function(add_test_without_ctest target)
         COMMAND $<TARGET_FILE:${target}> --gtest_output=xml:gtests-${target}.xml)
 endfunction()
 
+
+find_package(GMock REQUIRED)
+find_package(GTest REQUIRED)
+
+if(FALSE)
+
 # Build gmock
 set(gmock_build_tests           OFF CACHE BOOL "")
 set(gtest_build_samples         OFF CACHE BOOL "")
@@ -64,6 +70,7 @@ set(gtest_disable_pthreads      OFF CACHE BOOL "")
 set(gtest_force_shared_crt      ON  CACHE BOOL "")
 set(gtest_hide_internal_symbols OFF CACHE BOOL "")
 
+
 add_subdirectory(googletest/googlemock)
 
 # Create interface library to link against gmock
@@ -80,6 +87,7 @@ target_link_libraries(gmock-dev
     gmock
 )
 
+endif(FALSE)
 
 # 
 # Target 'test'
diff --git a/source/tests/cpplocate-test/CMakeLists.txt b/source/tests/cpplocate-test/CMakeLists.txt
index 56d103c..9c7ca3a 100644
--- a/source/tests/cpplocate-test/CMakeLists.txt
+++ b/source/tests/cpplocate-test/CMakeLists.txt
@@ -57,6 +57,8 @@ target_include_directories(${target}
     PRIVATE
     ${DEFAULT_INCLUDE_DIRECTORIES}
     ${PROJECT_BINARY_DIR}/source/include
+    ${GTEST_INCLUDE_DIR}
+    ${GMOCK_INCLUDE_DIR}
 )
 
 
@@ -68,7 +70,8 @@ target_link_libraries(${target}
     PRIVATE
     ${DEFAULT_LIBRARIES}
     ${META_PROJECT_NAME}::cpplocate
-    gmock-dev
+    ${GTEST_LIBRARIES}
+    ${GMOCK_LIBRARIES}
 )
 
 
diff --git a/source/tests/liblocate-test/CMakeLists.txt b/source/tests/liblocate-test/CMakeLists.txt
index 590d95a..69f10c8 100644
--- a/source/tests/liblocate-test/CMakeLists.txt
+++ b/source/tests/liblocate-test/CMakeLists.txt
@@ -61,6 +61,8 @@ target_include_directories(${target}
     PRIVATE
     ${DEFAULT_INCLUDE_DIRECTORIES}
     ${PROJECT_BINARY_DIR}/source/include
+    ${GTEST_INCLUDE_DIR}
+    ${GMOCK_INCLUDE_DIR}
 )
 
 
@@ -72,7 +74,8 @@ target_link_libraries(${target}
     PRIVATE
     ${DEFAULT_LIBRARIES}
     ${META_PROJECT_NAME}::liblocate
-    gmock-dev
+    ${GTEST_LIBRARIES}
+    ${GMOCK_LIBRARIES}
 )
 
 
